extensions/2.0/Vendor/DRACO_animation_compression/README.md

```diff
@@ -0,0 +1,236 @@
+                    }
+                },
+            ],
+            "samplers" : [
+                {
+                    "input" : 0
+                    "interpolation" : "LINEAR",
+                    "output" : 1
+                },
+                {
+                    "input" : 0
+                    "interpolation" : "LINEAR",
+                    "output" : 2
+                }
+            ]
+        },
+        {
+            "channels" : [
+                {
+                    "sampler" : 0,
+                    "target" : {
+                        "node" : 2,
+                        "path" : "rotation"
+                    }
+                },
+                {
+                    "sampler" : 1,
+                    "target" : {
+                        "node" : 3,
+                        "path" : "translation"
+                    }
+                },
+            ],
+            "samplers" : [
+                {
+                    "input" : 0
+                    "interpolation" : "LINEAR",
+                    "output" : 3
+                },
+                {
+                    "input" : 4
+                    "interpolation" : "LINEAR",
+                    "output" : 5
+                }
+            ]
+        },
+    ],
+    "accessors" : [
+        // Accessors of input/output of animations that use the compression extension will omit "bufferView" property.
+        {
+            "componentType" : 5123,
+            "count" : 100,
+            "type" : "SCALAR"
+        },
+        {
+            "componentType" : 5123,
+            "count" : 100,
+            "type" : "VEC3"
+        },
+        {
+            "componentType" : 5123,
+            "count" : 100,
+            "type" : "VEC4"
+        },
+        {
+            "componentType" : 5123,
+            "count" : 100,
+            "type" : "VEC4"
+        },
+        {
+            "componentType" : 5123,
+            "count" : 100,
+            "type" : "SCALAR"
+        },
+        {
+            "componentType" : 5123,
+            "count" : 100,
+            "type" : "VEC3"
+        },
+        ...
+    ],
+    "bufferViews" : [
+        // Contains data for the first compressed animation.
+        {
+            "buffer" : 10,
+            "byteOffset" : 0,
+            "byteLength" : 10000
+        },
+        // Contains data for the second compressed animation.
+        {
+            "buffer" : 10,
+            "byteOffset" : 10000,
+            "byteLength" : 6000
+        },
+        ...
+    ]
+}
+
+
+```
+
+### JSON Schema
+
+For full details on the `Draco_animation_compression` extension properties, see the schema:
+
+* [extension property (the entirety of this specification is normative and included in Scope)](schema/node.DRACO_animation_compression.schema.json)
+
+## Conformance
+
+Below is the recommended process when a loader encounters a glTF asset with the Draco animation extension set:
```

Comment:
The risk of calling this a "recommended process" is that it makes the actual requirements for exporters and clients implicit... can these be phrased as normative statements on the content and interpretation of the extension data, where possible?